### DISCUSSION TITLE
[Jetpack Compose] KmpAndroid initializeApp

### DISCUSSION THREAD

**Discussion by flipper09112** [Ares-Defence-Labs/KmpEssentials#2] [Q&A] [answered]
How its expected to initialize the lib when we have a MainActiity that is ComponentActivity?

**Comment by TheArchitect123** [comment, marked as answer]
I'm going to migrate to ComponentActivity because I had a screenshot service that requires AppCompatActivity for it to work. 
Originally I needed it for a commercial project I was working on. But I'm going to change the API for the screenshot service so I can support both Compose + Material Activities.

So I'm going to publish another release soon that will add support for Compose too.
Probably in a couple of days.

**Comment by TheArchitect123** [comment]
**Ok, Android Compose is now supported.**

You can install the latest package here.
`implementation("io.github.thearchitect123:kmpEssentials:0.8.6")`

**And you can find setup instructions for Android Compose here**
https://github.com/TheArchitect123/KmpEssentials/blob/main/README.md#how-to-use
